docs/writing-tests/file-names.md

```diff
@@ -29,6 +29,9 @@ themselves precede any test type flag, but are otherwise unordered.
 `.https`
  : Indicates that a test is loaded over HTTPS.
 
+ `.h2`
```

Comment:
Now this is adding a new test type which seems like a more major addition than just enabling by default and perhaps worthy of an RFC; when I said write tests I was more expecting something just loading something cross-origin and checking the server is responding (and given some of the other changes in the PR now, I wonder if it wasn't).

Reply:
The RFC is now merged.